### ISSUE TITLE
[Question] Is it posisble to get a tweet based on Twitter Status ID?

### ISSUE THREAD

**Issue by santoshbs** [Mottl/GetOldTweets3#100]
I was wondering if we could use GetOldTweets3 to fetch the tweet text for a given twitter status id or a set of status ids. I could not find this in the documentation.

**Comment by emilbakke** [comment]
I do not think this is possible. However, if you have a list of tweet IDs, you can feed them to tweepy and fetch tweets through the official Twitter API. Depending on how big your list is of couse, since there will be a timeout every ~900 tweets, I believe.
